tests/zzz_copy_sut_archive_files.rb

```diff
@@ -1,16 +1,15 @@
 require 'fileutils'
 
 step "Copy archive files from SUT" do
-  archive_files = ENV['SUT_ARCHIVE_FILES'].split(",")
-  job_name = ENV['PUPPET_GATLING_SIMULATION_ID']
-  Beaker::Log.notify("Copying #{archive_files.count} archive files from SUT")
-
-  archive_dir = "../puppet-gatling/#{job_name}/sut_archive_files"
-
-  FileUtils.mkdir_p(archive_dir)
-
-  archive_files.each do |s|
-    Beaker::Log.notify("Copying archive file '#{s}'")
-    scp_from(master, s, archive_dir)
-  end
+  today = Date.today().to_s
```

Comment:
I don't see date being required...  (anywhere in gatling repo at least)

Reply:
Date is part of core ruby, doesn't need to be required.

Reply:
Ok, so I was wrong: https://stackoverflow.com/questions/11631975/why-does-date-exist-in-ruby-before-it-is-required/11633934#11633934

Reply:
But, this does work, it must be required by one of the modules mixed in by Beaker.

Reply:
It is best practice to require it anyway... to show the dependency?  Ruby should be smart enough to do nothing if it is already required.   And that would make the code resilient in the case that for some reason whatever did require it stops doing so... like the beaker modularization effort that trys to load less things unless needed.